tests/testthat/_snaps/print_ard_conditions.md

```diff
@@ -13,9 +13,9 @@
           mean(x)
         }, err_fn = function(x) stop("'tis an error"))))
     Message
-      The following errors were returned while calculating statistics:
+      The following errors were returned during `print_ard_conditions()` and `ard_continuous(ADSL, variables = AGE, statistic = ~list(mean = function(x) mean(x), mean_warning = function(x) { warning("warn1") warning("warn2") mean(x) }, err_fn = function(x) stop("'tis an error")))()`:
```

Comment:
hmmmm, I don't think this should change. need to investigate

Reply:
This is caused by this section in the code - how do you think we should handle this?

https://github.com/insightsengineering/cards/blob/9177db996c20451c30dd6a2c4ac7e2b0d7380147/R/print_ard_conditions.R#L100

Reply:
I need to check/think more about the best way to handle this....

Reply:
ok i think this is handled. can you review the update and let me know what you think?

Reply:
Updated message looks much better!! :)